execution/gethexec/sequencer.go

```diff
@@ -76,10 +79,23 @@ type SequencerConfig struct {
 	ExpectedSurplusSoftThreshold string          `koanf:"expected-surplus-soft-threshold" reload:"hot"`
 	ExpectedSurplusHardThreshold string          `koanf:"expected-surplus-hard-threshold" reload:"hot"`
 	EnableProfiling              bool            `koanf:"enable-profiling" reload:"hot"`
+	Timeboost                    TimeboostConfig `koanf:"timeboost"`
 	expectedSurplusSoftThreshold int
 	expectedSurplusHardThreshold int
 }
 
+type TimeboostConfig struct {
+	Enable                bool          `koanf:"enable"`
+	ExpressLaneAdvantage  time.Duration `koanf:"express-lane-advantage"`
+	SequencerHTTPEndpoint string        `koanf:"sequencer-http-endpoint"`
+}
+
+var DefaultTimeboostConfig = TimeboostConfig{
+	Enable:                false,
+	ExpressLaneAdvantage:  time.Millisecond * 250,
```

Comment:
Is this initialized at 1 and then multiplied by 250 to achieve 250ms? If yes, shouldnt this be 200ms?